pyramid_openapi3/tests/test_contenttypes.py

```diff
@@ -0,0 +1,79 @@
+"""Test different request body content types."""
+
+from pyramid.config import Configurator
+from pyramid.request import Request
+from pyramid.router import Router
+from webtest.app import TestApp
+
+import tempfile
+import typing as t
+import unittest
+
+
+def app(spec: str, view: t.Callable, route: str) -> Router:
+    """Prepare a Pyramid app."""
+    with Configurator() as config:
+        config.include("pyramid_openapi3")
+        config.pyramid_openapi3_spec(spec)
+        config.add_route("foo", route)
+        config.add_view(openapi=True, renderer="json", view=view, route_name="foo")
+        return config.make_wsgi_app()
+
+
+OPENAPI_YAML = """
+    openapi: "3.0.0"
+    info:
+      version: "1.0.0"
+      title: Foo
+    components:
+      schemas:
+        ObjectWithName:
```

Comment:
This is inconsequential, but I would maybe rename this schema to `ObjectWithBar` or `BarObject`, since the object does not actually have a name anymore (it did have a `name` property in my draft PR).